### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_ColorContrast: The separator color contrasts is below 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#11194]
["Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11yTCS;#A11yWCAG2.1;#Benchmark;#DesktopApp;#WCAG1.4.11;#SQL Azure Data Studio;#FTP;#ColorContrast;#A11ySev2;#Acc123;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:WCAG1.4.11
Tool: Accessibility insights for desktop

**Repro Steps:**
1.Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook
4. Click on cell and select code cell navigate to toggle more and open it .
5. Launch Accessibility insights for desktop and select third tab contrast.

**Actual:**
When the color contrast of the separator under toggle more button is checked the ratio is 1.557:1

**Expected:**
The color contrast of the separator should be 3:1 or greater

**User Impact:**
Low vision users will face difficulty in viewing the separate sections in the lsit .

**Attachment link for Reference:**
![11194A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_ColorContrast](https://user-images.githubusercontent.com/24580570/86442642-bbc95a00-bd2b-11ea-9c06-d569eec0abf2.png)

**Comment by chlafreniere** [comment]
I've reached out to the accessibility team internally. I see the same behavior in a number of other apps:
- VS Code
- Edge (Chromium)
- Teams
- Notepad++

Waiting on a definitive answer from the desktop app accessibility SME, but this is the information I've been given so far:

"Electron does things a bit differently and because of the Chromium engine there are just things we can’t fix in the Electron framework that we can in other frameworks."

**Comment by chlafreniere** [comment]
We chatted about this in accessibility office hours, and decided the following:

![image](https://user-images.githubusercontent.com/40371649/90565215-691cf380-e15b-11ea-8b77-a953558c585b.png)

This is because there is more space between items that have the separator (rather than items that don't have the separator).

**Comment by fsteffi** [comment]
Closing the bug as per the above comment and logging the same bug as a usability bug

**Comment by Yashi1201** [comment]
As per above comment, closing the bug.
**GithubTags**:#Closed;
